### DISCUSSION TITLE
Troubles accessing files in PRs

### DISCUSSION THREAD

**Discussion by vojtechjelinek** [community/community#22538] [Pull Requests]
<p>Hey, in some PRs in one of the repositories that I’m the maintainer of (oppia/oppia), I have trouble accessing the files tab.</p>
<p>For example, for <a href="https://github.com/oppia/oppia/pull/14874" rel="noopener nofollow ugc">https://github.com/oppia/oppia/pull/14874</a> I receive a 500 HTTP error and for <a href="https://github.com/oppia/oppia/pull/14728" rel="noopener nofollow ugc">https://github.com/oppia/oppia/pull/14728</a> only around 10 files are loaded and the rest cannot be loaded. For other PRs everything seems to be working just fine, I only had a problem submitting comments to some other PR, but in the end, the comments were submitted successfully. Some other members of our community are also reporting similar problems (not sure if in the same or different PRs).</p>
<p>The issue is happening for around 3 hours now and as I didn’t see any info about it on the status tracker I’m creating this topic.</p>

**Comment by srijanreddy98** [comment]
<p>This is happening to me too!</p>

**Comment by sdirosa** [comment]
<p>Same here. Started a couple of hours ago and still ongoing for one of our repositories.</p>

**Comment by chiragbaid7** [comment]
<p>Same here. I have been facing this issue for around 6 hours now.</p>

**Comment by sdirosa** [comment, marked as answer]
<p>Seems like this has been resolved :pray:</p>
